doc/manuals/admin/cli.md

```diff
@@ -91,6 +91,7 @@ The list of available options is the following:
     - WARNING (show error and warning messages - this is the default setting),
     - INFO    (show error, warning and informational messages),
     - DEBUG   (show ALL messages).
+    Note that the log level can be modified in run-time, using the [admin API](management_api.md).
```

Comment:
Maybe in L88:

Select logging level, supported levels -> Select initial logging level, supported levels:

(Not sure which one is better, just to see other opinions...)